lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -111,19 +111,29 @@ def self.schema_with_allowed_link(property,
                  type: 'Date',
                  required: false
 
-          schema :estimated_time,
-                 type: 'Duration',
-                 required: false,
-                 writable: false
+          property :estimated_time,
+                   exec_context: :decorator,
+                   getter: -> (*) do
+                     representer = ::API::Decorators::PropertySchemaRepresenter
+                       .new(type: 'Duration',
+                            name: WorkPackage.human_attribute_name(:estimated_time))
+                     representer.writable = represented.estimated_time_writable?
+                     representer
+                   end
 
           schema :spent_time,
                  type: 'Duration',
                  writable: false
 
-          schema :percentage_done,
-                 type: 'Integer',
-                 title: WorkPackage.human_attribute_name(:done_ratio),
-                 writable: false
+          property :percentage_done,
```

Comment:
We should completely hide this property in the schema when the percentage done is disabled.

Because it does not exist, right?